src/datachannel.html

```diff
@@ -0,0 +1,119 @@
+      <option value="default">All Addresses</option>
+    </select>
+    <br>
+    <button id="sendMessageBtn">Send</button><br>
+  </div>
+
+  <script>
+    document.addEventListener('DOMContentLoaded', async () => {
+      // Import VCS_DOMAIN and ROOM_TOKEN from config.js.
+      // Edit config.js with your own vcs-domain and token.
+      await window.vcs(document);
+
+      const messageElement = document.getElementById('message');
+
+      let room;
+      let participantAddress;
+
+      const joinLeaveBtn = document.getElementById('joinLeaveBtn');
+      const videos = document.getElementById('videos');
+      const dropDownListElement = document.getElementById('participantDropDown');
+
+      sendMessageBtn.addEventListener('click', async () => {
+        data = messageElement.value;
+
+        participantAddress = dropDownListElement.options[dropDownListElement.selectedIndex].value;
+
+        // If no particular address is selected, send to all participants.
+        if (participantAddress === 'default') {
+          await room.sendMessageToParticipant(data);
+        } else {
+          await room.sendMessageToParticipant(data, participantAddress);
+        }
+
+        // Reset message Value to a blank slate.
+        message.value = '';
+      });
+
+      joinLeaveBtn.addEventListener('click', async () => {
+        if (!room) {
+          joinLeaveBtn.disabled = true;
+          room = await RealtimeSdk.joinRoom(ROOM_TOKEN, {
+            host: VCS_DOMAIN
+          });
+
+          joinLeaveBtn.disabled = false;
+          joinLeaveBtn.innerText = 'Leave';
+          render(room.localParticipant);
+          room.remoteParticipants.forEach(render);
+          room.on('participantJoined', render);
+          room.on('participantLeft', remove);
+          room.on('roomLeft', cleanup);
+
+          // Remote participant sees who sent the message
+          let participant;
+          for (const user in room.remoteParticipants) {
+            if (user.address === participantAddress) {
+              participant = user;
+            }
+          }
+
+          await room.on('messageReceived', (participant, data) => {
+            alert(`Participant ${participant.address} sent message: ${data}`);
+          });
+        } else {
+          room.leave();
+          room = null;
+          joinLeaveBtn.innerText = 'Join';
+        }
+      });
+
+      function render (participant) {
+        const div = document.createElement('div');
+        div.id = participant.address;
+        // Let SDK create the video element, and update changes to the media stream
+        participant.attach(div);
+        videos.appendChild(div);
+        // On Render, add new element to option
+        const select = document.getElementById('participantDropDown');
+        if (participant.address !== room.localParticipant.address) {
+          select.options[select.options.length] = new Option(participant.address, participant.address);
```

Comment:
Use the `add` API on the select element to add new options.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/add